option optimize_for = LITE_RUNTIME;

message OpenUIReport
{
 required uint32 ui_id = 1;
}
